lib/message/processMIME.js

```diff
@@ -49,8 +49,8 @@ const verifySignature = async ({ publicKeys = [], date }, data) => {
     const body = parts[1];
 
     const { data: subdata, verified, signatures } = await verifyMessage({
-        message: createCleartextMessage(body),
-        publicKeys,
+        message: await createMessage(body),
```

Comment:
Is this equivalent?

Reply:
No, it should probably stay as it was :sweat_smile: